### ISSUE TITLE
calligra fails to build

### ISSUE THREAD

**Issue by davidak** [NixOS/nixpkgs#172139]
### Describe the bug

```
error: builder for '/nix/store/6zz5p3x7gpc3gd689nfh0q8ggg25g5qm-calligra-3.2.1.drv' failed with exit code 2;
       last 10 log lines:
       > [ 67%] Building CXX object sheets/CMakeFiles/calligrasheetsodf.dir/SheetPrint_p.cpp.o
       > [ 67%] Building CXX object sheets/CMakeFiles/calligrasheetsodf.dir/odf/SheetsOdfValidity.cpp.o
       > [ 67%] Building CXX object sheets/CMakeFiles/calligrasheetsodf.dir/odf/SheetsOdfRegion.cpp.o
       > [ 67%] Building CXX object sheets/CMakeFiles/calligrasheetsodf.dir/odf/SheetsOdfCell.cpp.o
       > [ 67%] Building CXX object sheets/CMakeFiles/calligrasheetsodf.dir/Value.cpp.o
       > [ 67%] Building CXX object sheets/CMakeFiles/calligrasheetsodf.dir/ValidityStorage.cpp.o
       > [ 67%] Building CXX object sheets/CMakeFiles/calligrasheetsodf.dir/Validity.cpp.o
       > [ 67%] Linking CXX shared library libcalligrasheetsodf.so
       > [ 67%] Built target calligrasheetsodf
       > make: *** [Makefile:136: all] Error 2
       For full logs, run 'nix log /nix/store/6zz5p3x7gpc3gd689nfh0q8ggg25g5qm-calligra-3.2.1.drv'.
```

https://hydra.nixos.org/jobset/nixpkgs/trunk#tabs-jobs

![Screenshot from 2022-05-09 09-09-34](https://user-images.githubusercontent.com/91113/167358331-c7d7989c-7d84-46fb-9bb4-548cbac82216.png)

### Steps To Reproduce
1. ...
2. ...
3. ...

### Expected behavior
no fail

### Notify maintainers
@ebzzry @zraexy

### Metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.34, NixOS, 22.05 (Quokka), 22.05pre375061.c777cdf5c56`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.0`
 - channels(root): `"home-manager-21.11.tar.gz, nixos, nixos-hardware, nixos-unstable"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by vcunat** [comment]
From what I looked earlier, probably due to poppler update.  Here's similar stuff for inkscape: https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=ea7abbfe5d1f6fbe2f25979b00e3d36651e0a041

**Comment by andrevmatos** [comment]
Should be fixed by #172733
